projects/igniteui-angular/src/lib/grids/grouping/group-by-area.directive.ts

```diff
@@ -67,6 +67,14 @@ export abstract class IgxGroupByAreaDirective {
         this._dropAreaMessage = value;
     }
 
+    /**
+     * Emitted when grouping is performed.
+     *
+     * @example
+     * ```html
+     * <igx-grid-group-by-area (expressionsChange)="expressionsChange($event)"></igx-grid-group-by-area>
+     * ```
+     */
```

Comment:
This is kind of werid cus the `IgxTreeGridGroupByAreaComponent` which is mentioned here: https://www.infragistics.com/products/ignite-ui-angular/angular/components/treegrid/groupby is a hidden component

Reply:
reverting the description

Reply:
Should we also remove the hidden and set description of the `IgxTreeGridGroupByAreaComponent` since it is mentioned in our docs?

Reply:
Probably yes, but doesn't sound like a part of this efforts